Makefile

```diff
@@ -60,17 +60,19 @@ TARGETS += iptables
 TARGETS += ipxe
 TARGETS += kmod
 TARGETS += libaio
-TARGETS += libattr
```

Comment:
this looks like a bug in previous commits, but this should stay

Reply:
I did a make rekres. Should I just add `TARGETS += libattr` back?